### PR TITLE
Fixes #14296: System technique try to edit pg_hba file only on SUSE, but packaging takes care of this part

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/14296

### CODE REVIEW

File: techniques/system/server-roles/1.0/postgres-check.st [hunk 1]
```diff
@@ -1,69 +0,0 @@
...
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-#####################################################################################
-
-bundle agent root_postgres_check
-{
-
-  vars:
-
-    SuSE::
```
Comment (amousset) [inline techniques/system/server-roles/1.0/postgres-check.st]: Wasn't this an exception for SLES?

Comment (ncharles) [reply techniques/system/server-roles/1.0/postgres-check.st]: this is done here, on all rpm systems
https://github.com/Normation/rudder-packages/blob/branches/rudder/4.1/rudder-reports/SPECS/rudder-reports.spec#L163

Comment (amousset) [reply techniques/system/server-roles/1.0/postgres-check.st]: but do you know why there was an exception for SLES?

Comment (ncharles) [reply techniques/system/server-roles/1.0/postgres-check.st]: it's from the old time https://issues.rudder.io/issues/2207

**Comment by ncharles** [comment]
retargeting to 5.0

**Comment by ncharles** [comment]
PR replaced by https://github.com/Normation/rudder-techniques/pull/1424

**Pull request closed without merging**